src/components/Editable.js

```diff
@@ -220,6 +220,12 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
     }
   }, [isEditing, cursorOffset])
 
+  useEffect(() => () => {
+    if (contentRef.current) {
+      contentRef.current.blur()
+    }
+  })
```

Comment:
Can you verify that this is still needed, and if so, add a comment explaining why? I couldn't tell a difference with or without when I tested it, but of course I may have been missing a specific edge case.